debian/control

```diff
@@ -0,0 +1,42 @@
+Source: eos-data-distribution
+Section: python
+Priority: standard
+Maintainer: Philip Withnall <withnall@endlessm.com>
+Build-Depends:
+ debhelper (>= 10),
+ dh-python,
+ python-all,
+ python-pyndn,
+ python-pytest,
+ python-setuptools,
+ python-setuptools-git,
+ python3-all,
+ python3-pyndn,
+ python3-setuptools,
+ python3-setuptools-git,
+Standards-Version: 3.9.7
+Testsuite: autopkgtest-pkg-python
+X-Python-Version: >= 2.7
+X-Python3-Version: >= 3.0
+
+Package: python-eosdatadistribution
```

Comment:
We could fix this by splitting it out into things like `python-edd-utils`, `python-edd-stores`, `python-edd-producers`, etc. — but see my comment below.